src/EnsembleKalmanInversions.jl

```diff
@@ -152,17 +152,42 @@ function EnsembleKalmanInversion(inverse_problem; noise_covariance=1e-2, resampl
     return eki
 end
 
+"""
+    Φ(eki, θ, G)
+
+Return a tuple `(Φ1, Φ2)` of terms in the EKI regularized objective function, where
+Φ = (1/2)*(Φ1 + Φ2). Φ1 measures output misfit `|| Γy^(-¹/₂) * (y .- G(θ)) ||²` and 
+Φ2 measures prior misfit `|| Γθ^(-¹/₂) * (θ .- μθ) ||²`, where `y` is the observation 
+map, `G(θ)` is the forward map, `Γy` is the observation noise covariance, `Γθ` is 
+the prior covariance, and `μθ` represents the prior means. Note that `Γ^(-1/2) = 
+inv(sqrt(Γ))`. 
+"""
+function Φ(eki, θ::AbstractVector, G::AbstractVector)
```

Comment:
Why `Φ`?

Reply:
That's typically what it's referred to as in the EKI papers.

Reply:
What is "it"?

Reply:
The regularized least-squares objective function 

![image](https://user-images.githubusercontent.com/51460867/152555929-2096218a-8964-4820-955b-69c4c0d0fb7e.png)